libtsuba/src/RDG.cpp

```diff
@@ -556,6 +556,10 @@ tsuba::RDG::DoMake(
   std::vector<PropStorageInfo*> part_info =
       KATANA_CHECKED(core_->part_header().SelectPartitionProperties());
 
+  // these are not Node/Edge types but rather property types we are checking
+  KATANA_CHECKED(core_->EnsureNodeTypesLoaded(rdg_dir_));
+  KATANA_CHECKED(core_->EnsureEdgeTypesLoaded(rdg_dir_));
+
```

Comment:
I believe it would make the most sense for these to be after the call to `AddProperties()` on line 568? (might be what @tylershunt was referring to for future work? 

approving anyway to unblobk

Reply:
It has to be before the `if(part_info.empty())` on line 563. And I think the call to `AddProperties()` on 568 is just using the property loading logic to load partition metadata. So it won't touch any node or edge properties, which are the only things these two calls will touch.

Reply:
OH. Simon is totally right; apologies.